packages/drawer/src/react/index.js

```diff
@@ -125,6 +125,8 @@ class Drawer extends React.Component {
 
   render() {
     const { isOpen, context: { themeName } } = this
+    const ariaLabel = isOpen ? 'Collapse' : 'Expand'
```

Comment:
@dborodyansky, quick ux question, is "collapse" and "expand" the language you'd like to go with here?

Reply:
👍 Yes. Thank you.